ios/pod_helpers.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+def try_pod(name, podspec, project_root)
+  pod name, :podspec => podspec if File.exist?(File.join(project_root, podspec))
+end
```

Comment:
Just confirming my understanding; this helper exists in order to support the same RN version numbers but for different platforms with differing pods, right?

Reply:
Yup. In `react-native-macos`, Folly was renamed RCT-Folly. This hasn't been upstreamed yet. `react-native-macos` also bundles `boost-for-react-native.podspec`. iOS doesn't. It would be nice if we can remove this function the future 😉

Reply:
Totes; working on it! 😄 https://github.com/facebook/react-native/pull/29748/files